app/models/concerns/config/aca_model_concern.rb

```diff
@@ -48,12 +48,12 @@ def aca_shop_market_employer_contribution_percent_minimum
       @@aca_shop_market_employer_contribution_percent_minimum ||= Settings.aca.shop_market.employer_contribution_percent_minimum
     end
 
-    def aca_transmit_scheduled_employers
-      @@aca_aca_transmit_scheduled_employers ||= Settings.aca.transmit_scheduled_employers
+    def aca_shop_market_transmit_scheduled_employers
+      @@aca_aca_shop_market_transmit_scheduled_employers ||= Settings.aca.shop_market.transmit_scheduled_employers
```

Comment:
why is this named as "aca_aca"

Reply:
this was an existing change. anyways i update it